app/informativedialog.h

```diff
@@ -0,0 +1,23 @@
+#ifndef INFORMATIVEDIALOG_H
+#define INFORMATIVEDIALOG_H
+
+#include <QDialog>
+
+namespace Ui
+{
+class InformativeDialog;
+}
+
+class InformativeDialog : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit InformativeDialog(QString title, QString description = QString(), QWidget *parent = 0 );
+    ~InformativeDialog();
+
+private:
+    Ui::InformativeDialog *ui;
+};
```

Comment:
This is somewhat true, although the idea with a custom dialog box is that you can do whatever you want. A QMessageBox is fixed to whatever settings it has. For example having the "ok" button to the right.

My goal is to make our own unique UI scheme that fits Pencil when we've fixed the tools and other things with higher priority, thus having standard buttons, toggles and such has to go.